### DISCUSSION TITLE
Reolink RLC-410W Camera has a green screen

### DISCUSSION THREAD

**Discussion by Kate222** [blakeblackshear/frigate#1244] [Ask A Question] [answered]
My Reolink RLC-410W Camera  has a green screen now. I have not changed anything in frigate.yml file. However, I have updated HA core recently. My other two cameras with frigate are working fine.

**Comment by blakeblackshear** [comment]
I haven't heard anyone else reporting this issue. Please open an issue and provide the information requested in the issue template.

**Reply by Kate222** [reply]
I guess not many people use rpi+ha+frigate with this camera. I can not view this camera through motioneye as well. I am not sure it  is worth to spend your valuable time on it.

**Reply by blakeblackshear** [reply]
I would open an actual issue and provide the info requested in the template. You haven't provided enough information for me to suggest anything.

**Reply by rhatguy** [reply]
I have 4*RLC-410 (non W model).  I assume they operate similarly to the W model though.  I also use an RPI4, HA, and frigate.  They can definitely be finicky, but I've got mine working pretty stable.  You might also need to add a few configurations on the RPI such as the gpu_mem config in /boot/config.txt.  As Blake said...post your configs and you might get more help.  I'd be fairly confident that the RLC-410W will work though.

**Comment by Kate222** [comment, marked as answer]
After a week of green screen, my camera is now back to normal again.
